docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: "3.0"
+
+services:
+  server:
+    build:
+      context: .
+    restart: always
+    ports:
+      - "7171:7171"
+      - "7172:7172"
+    depends_on:
+      - db
+    networks:
+      - db-network
+  db:
+    image: mysql
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: <your_root_password>
```

Comment:
Try `MYSQL_RANDOM_ROOT_PASSWORD: '1'`

Reply:
I would use `MYSQL_RANDOM_ROOT_PASSWORD` instead. It does not need a root account at all, and the less privileges, the better.